lib/cvc.ml

```diff
@@ -237,7 +237,7 @@ struct
       n: Derivable.Z.t;
       e: Derivable.Z.t;
     }
-    [@@deriving ord,eq,yojson,eq,show,bin_io]
+    [@@deriving ord,eq,eq,show]
```

Comment:
Is there a reason `eq` is listed twice? (Here and also below.)

Reply:
Fixed. Thanks!